docs/how-to-use-monty/code-base-structure.md

```diff
@@ -10,10 +10,9 @@ The basic repository structure looks as follows:
 .
 |-- docs/                       # .md files for documentation
 |-- rfcs/                       # Merged RFCs
-|-- benchmarks/                 # experiments testing Monty
-|   |-- configs/
-|   |-- run_parallel.py
-|   `-- run.py
+|-- conf/                       # Monty configurations
+|   |-- experiment/
+|   `-- test/
```

Comment:
I think there is also a `benchmarks` subfolder here (although, I think it just contains defaults so maybe you left it out here for brevity?)